src/app/visa/c.spec.ts

```diff
@@ -0,0 +1,630 @@
+// write test cases for visa C
+import { calculatePoints, errorMessages } from '@app/domain'
+
+import {
+    academicBackgroundWith,
+    annualSalaryOf,
+    contractingOrganizationOf,
+    japanese,
+    professionalCareerWith,
+    simulationWithCriteriaC,
+    specialOf,
+    universityOf,
+    positionInCompany
+} from '@app/spec.helper'
+
+// non university holder business owner
+// investment banker
+describe('Visa type C point simulation', () => {
+
+    describe('categories', () => {
+        describe('academic background', () => {
+            it('holder of multiple degrees in many areas', () => {
```

Comment:
This test case is nearly the same as `a dual degree should give bonus`. For this test case I'd just stick to a single degree as that is not covered yet.